compiler/rustc_privacy/src/errors.rs

```diff
@@ -55,9 +55,12 @@ pub struct InPublicInterface<'a> {
     pub span: Span,
     pub vis_descr: &'static str,
     pub kind: &'a str,
+    pub vis_sugg: &'static str,
     pub descr: DiagnosticArgFromDisplay<'a>,
     #[label(privacy_visibility_label)]
     pub vis_span: Span,
+    #[suggestion(code = "", applicability = "maybe-incorrect")]
```

Comment:
The suggested visibility should probably go to the `code` here, rather than to the suggestion message.